changelog.d/20240318_154804_kyle_assets_build.md

```diff
@@ -0,0 +1,16 @@
+- 💥[Feature] The `openedx-assets` command is replaced with `npm run` subcommands.
+  This will slightly reduce build time for edx-platform assets and themes.
+  It will also open up the door for more significant build time reductions in the future.
+  Here is a migration guide, where each command is to be run in the `lms` or `cms` container.
+  | **Before**                               | **After**                                                                           |
+  |------------------------------------------|-------------------------------------------------------------------------------------|
+  | `openedx-assets build  --env=prod ARGS`  | `npm run build -- ARGS`                                                             |
```

Comment:
This is very clear for people who are familiar with Tutor's openedx-assets comand. Is there a reference document that explains all these new npm commands? Unfortunately we can't add comments to [package.json](https://github.com/openedx/edx-platform/blob/master/package.json) because it's JSON.

Reply:
There isn't yet, but I'll add a reference doc to edx-platform and link it here.

Reply:
done!